docs/Project.toml

```diff
@@ -7,6 +7,7 @@ Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 HomotopyContinuation = "f213a82b-91d6-5c5d-acf7-10f1c761b327"
 Latexify = "23fbe1c1-3f47-55db-b15f-69d7ec21a316"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
```

Comment:
Did you mean to add this depedency? If so, it needs a compat too... Otherwise please remove.

Reply:
I checked, but I think this is a standard library dependency, and we don't have compats for these (right?).

(it is required because SI uses it to turn off the print out messages, which spam the docs if we have them on)